operator/pkg/util/util_test.go

```diff
@@ -0,0 +1,28 @@
+package util
+
+import (
+	"log"
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestListFileWithSuffix(t *testing.T) {
+	if os.Getenv("GITHUB_ACTIONS") == "true" {
```

Comment:
Does this mean this test can only run in GitHub workflow or a machine that sets the environment?

Reply:
Have created a test files at folder testdata, runtest always.